c2corg_ui/templates/image/filters.html

```diff
@@ -0,0 +1,68 @@
+<%!
+from c2corg_common.attributes import default_langs, activities, image_categories, image_types
+%>
+<%namespace file="../helpers/filters.html" import="add_multiselect"/>
+
+<form app-search-filters app-search-filters-controller-name="appSearchFiltersController">
+  <div class="filters">
+    <div class="row simple-filters">
+
+      <div class="filter">
+        <label translate>date_taken</label><br>
+        <div class="input-group" ng-model="filtersCtrl.dates[0]" datepicker-options="{maxDate: filtersCtrl.dateMaxStart}"
+             uib-datepicker-popup="dd MM yyyy" is-open="openDateStart" ng-change="filtersCtrl.setDate('date')">
+          <input type="text" disabled class="form-control" value="{{filtersCtrl.dates[0] | amDateFormat:'Do MMMM YYYY'}}"/>
+          <span class="input-group-btn">
+            <button type="button" class="btn btn-default" ng-click="openDateStart = true"><span class="glyphicon glyphicon-calendar"></span></button>
+          </span>
+        </div>
+      </div>
+
+      <div class="filter">
+        <label translate>activities</label><br>
+        ${add_multiselect('act', activities)}
+      </div>
+
+      <div class="filter">
+        <label translate>categories</label><br>
+        ${add_multiselect('cat', image_categories)}
+      </div>
+
+    </div>
+
+    <div class="row collapse" id="moreFilters">
+
+      <div class="col-xs-12 col-sm-6 filter">
+        <label translate>title</label>
+        <input type="text" class="form-control" ng-model="filtersCtrl.filters.q">
+      </div>
+
+      <div class="col-xs-12 col-sm-6 filter">
+        <label translate>elevation</label><br>
```

Comment:
Searching on elevation for images is very rare. I think we can remove this filter.